AutoGraph/WebSocketClient.swift

```diff
@@ -298,6 +325,41 @@ extension WebSocketClient: WebSocketDelegate {
         }
     }
     
+    func processWebSocketTextEvent(text: String) throws {
+        guard let data = text.data(using: .utf8) else {
+            throw ResponseSerializerError.failedToConvertTextToData(text)
+        }
+        
+        let json = try JSONValue.decode(data)
+        guard case let .string(typeValue) = json["type"] else {
+            return
+        }
+        
+        let graphQLWSProtocol = GraphQLWSProtocol(rawValue: typeValue)
+        switch graphQLWSProtocol {
+        case .connectionAck:
+            self.isConnectionAck = true
+            self.connectionAckWorkItem?.cancel()
+            self.connectionAckWorkItem = nil
+        case .data:
+            let subscriptionResponse = try self.subscriptionSerializer.serialize(text: text)
+            self.didReceive(subscriptionResponse: subscriptionResponse)
+        case .connectionKeepAlive:
+            self.writeQueuedSubscriptions()
+        case .error:
+            throw ResponseSerializerError.webSocketError(text)
+        case .complete:
+            guard case let .string(id) = json["id"] else {
+                return
+            }
+            
+            self.subscriptions.removeValue(forKey: id)
+            self.queuedSubscriptions = self.queuedSubscriptions.filter { $0.key.subscriptionID == id }
+        default:
+            break
```

Comment:
nope, we want to handle all cases. if you're doing this to handle binary later then instead handle that and break and add a `// TODO:`

Reply:
oh, i'm mixing protocol with events. still need to display all event cases here so we know they're probably handled, default should almost never be used in production code.